test/Base_Tests/src/Semantic/Meta_Location_Spec.enso

```diff
@@ -14,15 +16,15 @@ add_specs suite_builder = suite_builder.group "Meta-Value Inspection" group_buil
 
     group_builder.specify "should allow to get the source location of a frame" pending=location_pending <|
         src = Meta.get_source_location 0
-        loc = "Meta_Location_Spec.enso:16:15-40"
+        loc = "Meta_Location_Spec.enso:18:15-40"
         src.take (Last loc.length) . should_equal loc
 
     group_builder.specify "should allow to get qualified type names of values" <|
         x = 42
         y = My_Type.Value 1 2 3
         Meta.get_qualified_type_name x . should_equal "Standard.Base.Data.Numbers.Integer"
         Meta.get_simple_type_name x . should_equal "Integer"
-        Meta.get_qualified_type_name y . should_equal "enso_dev.Base_Tests.Semantic.Meta_Location_Spec.My_Type"
+        Meta.get_qualified_type_name y . should_end_with "Meta_Location_Spec.My_Type"
```

Comment:
Unrelated change. It ensures that one can run this test as standalone with `enso --run test/Base_Tests/src/Semantic/Meta_Location_Spec.enso`

Reply:
Won't the tests below break if run without `--in-project` anyway?

Reply:
There is no need to specify `--in-project` anymore since https://github.com/enso-org/enso/pull/8775